packages/vuetify/src/components/VCombobox/VCombobox.tsx

```diff
@@ -477,7 +477,8 @@ export const VCombobox = genericComponent<new <
         slots['append-item'] ||
         slots['no-data']
       )
-      const isDirty = model.value.length > 0
+      const isEmptyString = model.value.length === 1 && model.value[0].value === ''
```

Comment:
It should rather be named `isArrayWithEmptyString`. Regardless... I don't think a combobox with `multiple` and `model-value="['']"` can be considered empty. It behaves differently when selecting more values. Showing placeholder in this case could be misleading.